components/locale/vi_VN.tsx

```diff
@@ -41,6 +41,71 @@ const localeValues: Locale = {
   Empty: {
     description: 'Trống',
   },
+  Icon: {
+    icon: 'biểu tượng'
+  },
+  Text: {
+    edit: 'Sửa',
+    copy: 'Sao chép',
+    copied: 'Đã sao chép',
+    expand: 'Mở rộng'
+  },
+  PageHeader: {
+    back: 'Quay lại'
+  },
+  Form: {
+    optional: '(tuỳ chọn)',
+    defaultValidateMessages: {
+      default: 'Lỗi xác thực cho trường ${label}',
+      required: 'Hãy nhập ${label}',
+      enum: '${label} phải là mộ trong [${enum}]',
+      whitespace: '${label} không thể là một ký tự trống',
+      date: {
+        format: '${label} định dạng ngày không hợp lệ',
+        parse: '${label} không thể chuyển đổi sang định dạng ngày',
+        invalid: '${label} là một ngày không hợp lệ',
+      },
+      types: {
+        string: typeTemplate,
+        method: typeTemplate,
+        array: typeTemplate,
+        object: typeTemplate,
+        number: typeTemplate,
+        date: typeTemplate,
+        boolean: typeTemplate,
+        integer: typeTemplate,
+        float: typeTemplate,
+        regexp: typeTemplate,
+        email: typeTemplate,
+        url: typeTemplate,
+        hex: typeTemplate
+      },
```

Comment:
@tangjinzhou hey bro, this pr will cause a compiling error. `typeTemplate` is not defined.